scripts/loadtest-scripts/Makefile

```diff
@@ -4,7 +4,7 @@ help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 install-dev-dependendencies: ## Install the development dependencies
-	pipenv install --dev
+	pipenv install --dev --deploy
```

Comment:
🤔 dev and deploy...? Is this to ensure the lockfile hasn't changed?

Reply:
I knew someone was going to pick me up on that 😄 The intention was indeed to ensure the lockfile hasn't changed, but it does look a bit weird - maybe there is another way to achieve this?

Reply:
No no. If that's the intention then this is probably the best way.

Reply:
👍 - thanks